test/ui-test/extensionUITest.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+/* eslint-disable func-names */
```

Comment:
Please remove this, just change to arrow functions. `async function () {` to `async () => {`